CSharpSourceCode/SubModule.cs

```diff
@@ -68,6 +68,82 @@ protected override void OnSubModuleLoad()
             //ref https://forums.taleworlds.com/index.php?threads/ui-widget-modification.441516/ 
             UIConfig.DoNotUseGeneratedPrefabs = true;
             LoadFontAssets();
+            LoadDismembermentSettings();
+        }
+        protected override void OnApplicationTick(float dt)
```

Comment:
As mentioned above, you should be using a MissionLogic and override its OnMissionTick() method for this. Application Tick is not the place for this. Added benefit if you do it that way is that while in a Mission, you can be sure that Game.Current and Mission.Current exists guaranteed, no need for NPE check.